### PR TITLE
work around shrinked resources

### PR DESCRIPTION
**Pull request by dsvoronin**

### CODE REVIEW

**Comment by dsvoronin** [comment]
slackhq/keeper/issues/22

File: docs/content/test/TestMinimized.md [hunk 1]
```diff
@@ -24,3 +24,30 @@ However we could miss some bugs, because debug(not release) sources included, so
 ## Sample
 
 You can check configuration in `:test-app` module.
+
+## If you still have problems 
+
+Keeper can't do all the work here, so if you still can't run test, check these:
```
Comment (sergioboi) [inline docs/content/test/TestMinimized.md]: Keeper can't do everything. Know issues:

File: docs/content/test/TestMinimized.md [hunk 2]
```diff
@@ -24,3 +24,30 @@ However we could miss some bugs, because debug(not release) sources included, so
...
+
+### Dynamically referenced resources
+
+You can see in `PageObjectTest`, that we referenced `R.layout.page_object_1`, dynamically creating layout.\
```
Comment (sergioboi) [inline docs/content/test/TestMinimized.md:32]: Enabled shrinkResourses delete resourses referenced only from tests.
Example in PageObjectTest.

File: docs/content/test/TestMinimized.md (hunk 1 above)
Comment (dsvoronin) [reply docs/content/test/TestMinimized.md]: removed whole phrase

File: docs/content/test/TestMinimized.md (hunk 2 above)
Comment (dsvoronin) [reply docs/content/test/TestMinimized.md:32]: left as it was, i can hardly parse new variant

File: docs/content/test/TestMinimized.md [hunk 3]
```diff
@@ -24,3 +24,66 @@ However we could miss some bugs, because debug(not release) sources included, so
...
+```
+
+If this is your case, add these resources to `res/raw/keep.xml` like this:
+
+```xml
+<resources xmlns:tools="http://schemas.android.com/tools"
+           tools:keep="@layout/page_object*"/>
+```
+
+### Try latest stable version of R8
```
Comment (dsvoronin) [inline docs/content/test/TestMinimized.md:53]: "Already has a mapping"

Comment (eugene-krivobokov) [inline docs/content/test/TestMinimized.md:53]: **the** latest 

https://coub.com/view/1847e7

**Review by eugene-krivobokov — APPROVED** [review]

**Pull request merged**